src/lib.rs

```diff
@@ -28,8 +28,12 @@ use graphcast_sdk::{
 use crate::metrics::{CACHED_MESSAGES, VALIDATED_MESSAGES};
 
 pub mod attestation;
+pub mod graphql;
 pub mod metrics;
 
+pub type MessagesVec = OnceCell<Arc<SyncMutex<Vec<GraphcastMessage<RadioPayloadMessage>>>>>;
+pub type MessagesArc = Arc<SyncMutex<Vec<GraphcastMessage<RadioPayloadMessage>>>>;
```

Comment:
What is the reason we want to keep a `MessagesArc` instead of reading from `MessagesVec`?